### DISCUSSION TITLE
What does the "Partial Match" mean when verifying contracts

### DISCUSSION THREAD

**Discussion by yarkinwho** [blockscout/blockscout#10422] [Q&A] [answered]
The blockscout is configured to run with the microservice for contract verification. But what ever i verify, i always got Partial Match in the frontend. 
The question is:
1 what does that partial match mean
2 what can be wrong to have this pattern?
3 any suggestion to debug that

Thanks.

**Comment by sl1depengwyn** [comment, marked as answer]
1. https://docs.sourcify.dev/docs/full-vs-partial-match/
2. This is actually nothing wrong, partial match means verified but not with exact compilation artifacts
3. Use standard json verification instead of flattened code one if you want to get full match
